src/script/ceph-release-notes

```diff
@@ -218,7 +218,7 @@ def make_release_notes(gh, repo, ref, plaintext, verbose, strict, use_tags):
                        issue + " " + str(prs))
 
     for (pr, (author, title, message)) in sorted(
-        pr2info.items(), key=lambda k, v: (v[2], v[1])
+        pr2info.items(), key=lambda title: title[1][1]
```

Comment:
if we want to sort by <message,title> as before, probably, should be
```
key=lambda info: info[1][2], info[1][1]
```

in this change we are ordering by title solely.